sopel_waifu/waifu.json5

```
@@ -8609,6 +8609,7 @@
         "Sarah",
         "Sweety",
         "Colonel Isolde",
+        "Magus Magnusson", // Technically playable.
```

I don't get it. If playable, why not under the main (playable agents) section?

Nit:

```suggestion
        "Magus Magnusson",  // Technically playable.
```